store/store.go

```diff
@@ -123,6 +123,17 @@ func (a *Alerts) Delete(fp model.Fingerprint) error {
 	return nil
 }
 
+// DeleteIf removes the Alert with the matching fingerprint from the store
+// if fn returns true.
+func (a *Alerts) DeleteIf(fp model.Fingerprint, fn func(*types.Alert) bool) error {
+	a.Lock()
+	defer a.Unlock()
+	if alert, ok := a.c[fp]; ok && fn(alert) {
+		delete(a.c, fp)
+	}
+	return nil
```

Comment:
Do you have plans on returning an error in the near future?

Reply:
Nope! TBH I just added it because `Delete` has the same signature, even though it never returns an error:

```go
func (a *Alerts) Delete(fp model.Fingerprint) error {
```